packages/components/src/components/hds/side-nav/index.ts

```diff
@@ -27,6 +27,7 @@ interface HdsSideNavSignature {
     onToggleMinimizedStatus?: (arg: boolean) => void;
     // eslint-disable-next-line @typescript-eslint/no-explicit-any
     onDesktopViewportChange?: (arg: boolean) => void;
+    hasHeader?: boolean;
```

Comment:
Does this arg name make sense? This is meant to set whether the `SideNav is being accompanied by the `AppHeader`.

Would `hasAppHeader` or another name be clearer?

Reply:
Personally I think that `hasHeader` is more clear because it's specific to the component the argument is being invoked on.

Reply:
The `hasHeader` arg is used to alter the height and top position of the SideNav so it's visually aligned together with the fixed AppHeader. The name of the arg could possibly be mistaken to be related to the SideNav :header block is what I'm worried about. (When the SideNav is paired with the AppHeader, it's true that its :header block is meant to be left empty but the `hasHeader` arg is unrelated to that.)

```
.hds-side-nav--has-header {
  top: var(--token-hds-app-header-height);
  height: calc(100vh - var(--token-hds-app-header-height));
  min-height: calc(100vh - var(--token-hds-app-header-height));
}
```

Reply:
@didoo what do you think?

Reply:
I'm going to change the name of this parameter to `@withAppHeader`. I think that will make it's purpose & usage clearer without being too long.